public/Lab-Files/Lesson02/Habitat/CCYP/syb.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="keywords" content="CIW, HTML5, Habitat for Humanity"/>
+<meta name="description" content="Simple XHTML page for Habitat site"/>
+<meta charset="UTF-8"/>
+<title>Habitat for Humanity International Summer Youth Blitz Program</title>
+</head>
+<body style="background-color:white">
+
+<h1>Join a Summer Build for Teenagers</h1>
+<blockquote>
+<p>
+<em>The Summer Youth Blitz</em> is a unique service experience for a diverse group of
+youth,<br/>ages 16 to 18,
+from high schools and youth organizations around the United States.
+<br/>During this program, 15 to 20
+youth participants and adult leaders <br/>"blitz build" an entire Habitat
+house in two weeks.
+</p>
+</blockquote>
+
+<blockquote>
+<p>
+The house build, an unfamiliar activity for most, provides a common,
+<br/>nonthreatening ground for building
+relationships and teams.
+</p> 
+<!-- Bulleted list removed by request of the lab exercise, Wesley Supnet
+<p>During the house build, you will:</p>
+<ul>
+<li>Help lay the foundation. </li>
+<li>Assist in framing the home. </li>
+<li>Do simple carpentry, under supervision. </li>
+</ul> 
+-->
+</blockquote>
+<blockquote>
+<p>
+In the evenings, the youth participate in activities like team-building games,
+<br/>leadership development,
+local cultural events or community dinners.
+</p>
+</blockquote>
+<h2>Sponsors</h2>
+<blockquote>
+<p>
+This program is sponsored by national grant donations and coordinated by the
+Campus Chapters<br/> and Youth
+Programs department of Habitat for Humanity International. For the past
+several years, <br/>funding has allowed
+for three blitz builds per summer--two in the United States and one outside
+<br/>the country. The builds take
+place during the months of June, July and August.
+</p>
+</blockquote>
+<h3>Apply Now!</h3>
+<blockquote>
+<p>
+If you're interested in participating, <strong>and are between the ages of 16 and 18
+years old,</strong> <br/>you must submit
+an application, which is available during January and February.
+</p> 
+</blockquote>
+<p style="text-align:center"> For more information, contact us at (800) 422-
+4828, ext. 2220. </p> 
+
+<!-- Wesley Supnet, 03/09/17. This code validates to HTML5--> 
```

Comment:
Wesley,

The lab is looking good.

The only thing I'd suggest is working on indentation and spacing.

Nesting and indenting elements will make your document look clean and organized. It'll make it much easier to go back and edit content too.